src/Host/Client/Impl/BrokerServices/RemoteUriWebService.cs

```diff
@@ -22,17 +23,18 @@ public class RemoteUriWebService : IRemoteUriWebService {
 
         private Uri PostUri { get; }
 
-        public async Task GetResponseAsync(HttpListenerContext context, string localBaseUrl, string remoteBaseUrl, CancellationToken ct) {
+        public async Task GetResponseAsync(HttpListenerContext context, string localBaseUrl, string remoteBaseUrl, string webSocketsScheme, CancellationToken ct) {
             string postUri = null;
             if (context.Request.IsWebSocketRequest) {
-                UriBuilder ub = new UriBuilder(PostUri) { Scheme = "ws" };
+                UriBuilder ub = new UriBuilder(PostUri) { Scheme = webSocketsScheme };
```

Comment:
This is for ?help

---
In reply to: [82295312](https://github.com/Microsoft/RTVS/pull/2447#discussion_r82295312) [](ancestors = 82295312)